openreview/journal/process/author_submission_process.py

```diff
@@ -19,7 +19,7 @@ def process(client, edit, invitation):
 
 To do so, please follow this link: https://openreview.net/invitation?id={journal.get_ae_recommendation_id(number=note.number)} or check your tasks in the Author Console: https://openreview.net/group?id=.TMLR/Authors
 
-For more details and guidelines on the TMLR review process, visit jmlr.org/tmlr.
+For more details and guidelines on the TMLR review process, visit {journal.website}.
```

Comment:
all of this was done in the first PR: https://github.com/openreview/openreview-py/pull/1058

I didn't want to add both changes to the same PR since this affected the request form as well. I see there are some conflicts in the other PR which I'll fix now.

Reply:
Ok, I will merge #1058 first. Then we can merge this one, I think this PR shouldn't affect the beta testing, right? because the request form is being used for the recruitment and that is not being tested.

Reply:
No it should not affect beta testing, but we can edit the TMLR request form to include the website as well.